src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/pbs/PBSCommand.groovy

```diff
@@ -215,16 +214,20 @@ class PBSCommand extends Command {
         return qsubCall
     }
 
+    // TODO Code duplication with PBSCommand. Check also DirectSynchronousCommand.
     String assembleVariableExportString() {
         StringBuilder qsubCall = new StringBuilder()
 
+        if (job.parameters.containsKey("CONFIG_FILE") && job.parameters.containsKey("PARAMETER_FILE")) {
```

Comment:
Before we start to pull out the Roddy specific part, we definitely need to clean up and unify code first!

Reply:
This is the site I meant in our conversation.